### ISSUE TITLE
Align vscode views contents with header file

### ISSUE THREAD

**Issue by akosyakov** [eclipse-theia/theia#4637]
Follow-up of #4622 

You can reproduce with https://github.com/akosyakov/vscode-trees (needs #4624):
```bash
mkdir plugins && /
cd plugins && /
git clone https://github.com/akosyakov/vscode-trees.git && /
cd vscode-trees && /
npm i && /
npm run compile
```

How it looks now:
<img width="371" alt="Screen Shot 2019-03-21 at 11 38 54" src="https://user-images.githubusercontent.com/3082655/54747097-38373880-4bce-11e9-95ed-001deafcb03d.png">

How it looks in VS Code: 
https://github.com/akosyakov/vscode-trees/blob/master/resources/package-explorer.png

cc @jbicker

**Comment by vince-fugnitto** [comment]
@akosyakov I believe it's been addressed? I tested against the latest master 1808d78

![image](https://user-images.githubusercontent.com/40359487/64963723-bc116b00-d867-11e9-8f7c-520338701aa2.png)

**Comment by akosyakov** [comment]
@vince-fugnitto yes, thank you for the remainder!
